src/cmd/sign.js

```diff
@@ -117,6 +155,8 @@ export function getIdFromSourceDir(sourceDir: string): Promise<string> {
 }
 
 
+// getIdFromSourceDir helper implementation.
```

Comment:
The one below is wrong for sure (it refers to the wrong function name).

I added them mostly to separate this helpers from the above group more clearly, but I agree that using just the "2 blank line" as a separator can be more then enough.

I'm going to remove these comments (the ones that are before an helper without any type defined)